src/core/Microsoft.PowerToys.Settings.UI.Lib/BoolProperty.cs

```diff
@@ -17,6 +17,11 @@ public BoolProperty()
             this.Value = false;
         }
 
+        public BoolProperty(bool value)
+        {
+            Value = value;
+        }
+
         [JsonPropertyName("value")]
         public bool Value { get; set; }
 
```

Comment:
Instead of all this `TypeProperty` classes, couldn't we just use the `GenericProperty` class?